integration/test-integration/src/test/resources/testng-cc-standalone.xml

```diff
@@ -46,7 +46,6 @@
             <class name="org.wso2.choreo.connect.tests.testcases.standalone.endpoints.BackendSecurityTestCase"/>
             <class name="org.wso2.choreo.connect.tests.testcases.standalone.jwtValidator.JwtTestCase"/>
             <class name="org.wso2.choreo.connect.tests.testcases.standalone.jwtValidator.ScopeTest"/>
-            <class name="org.wso2.choreo.connect.tests.testcases.standalone.jwtValidator.InternalKeyTestCase"/>
```

Comment:
We cannot remove this testcase as it is a legitimate one, (which we use to test testKey in current choreo context)